src/Proto3/Suite/Form/Encode.hs

```diff
@@ -140,17 +144,17 @@ instance (omission ~ 'Alternative) =>
 -- | Precomputes the octet sequence that would be written by the given 'MessageEncoder'.
 -- Do this only if you expect to reuse that specific octet sequence repeatedly.
 --
--- @'cachedMessageEncoding' . 'cacheMessageEncoding'@ is functionally equivalent to @id@
--- but has different performance characteristics.
+-- @'cachedMessageEncoding' . 'cacheMessageEncoding'@ is functionally equivalent
+-- to @id@ but has different performance characteristics.
```

Comment:
`s/@id@/'id'` so that Haddock links to [`id`](https://hackage.haskell.org/package/base-4.21.0.0/docs/Prelude.html#v:id)?

Reply:
Sure; implemented by commit 3799856e2330c1aaba058d2d79c776d612f441db .